src/Web/Slack/Types.hs

```diff
@@ -25,7 +25,6 @@ module Web.Slack.Types
 import Data.Aeson
 
 -- base
-import Data.Monoid
```

Comment:
Are the `import Data.Monoid` only redundant when using newer versions (will this stop working for older versions)?

Reply:
yes absolutely. on the other hand, if we want to keep them due to old versions, we might want to think about #ifdef pragmas, or to disable -Wunused-imports. This is almost an ancient thing otherwise, by haskell standards. https://ghc.haskell.org/trac/ghc/wiki/Migration/7.10 -- This import is redundant since ghc 7.10 (base 4.8). Seems to be 2015. https://www.haskell.org/ghc/download_ghc_7_10_1.html -- you are correct though, if we want to drop that support, we could indicate it in the cabal file through constraints (concretely, base>=4.8).

So I guess:

1. drop -Wunused-imports
2. #if for the import on the `base` version to support both versions without warnings
3. require base >=4.8 (2015)

Reply:
I'm okay with 3, require `base >= 4.8`.

Reply:
hmm wait, we already have `base >= 4.9` in the .cabal so this seems like a non-issue :) in that case the PR is correct, we never need this import given the build constraints.